fvtest/threadtest/createTest.cpp

```diff
@@ -514,7 +514,7 @@ initDefaultExpected(create_attr_t *expected)
 	expected->priority = J9THREAD_PRIORITY_NORMAL;
 	expected->policy = J9THREAD_SCHEDPOLICY_INHERIT;
 
-	expected->osStacksize = getOsStacksize((size_t)expected->stacksize);
+	expected->osStacksize = getOsStacksize(expected->stacksize);
```

Comment:
The data type of the `osStacksize` field in the `create_attr_t` data struct will need to be changed to `uintptr_t`. Currently, the `osStacksize` field is of data type `size_t`.